common4j/src/main/com/microsoft/identity/common/java/commands/parameters/nativeauth/AcquireTokenNoFixedScopesCommandParameters.java

```diff
@@ -63,17 +63,91 @@ public class AcquireTokenNoFixedScopesCommandParameters extends BaseNativeAuthCo
     @Expose()
     private final Set<String> scopes;
 
+    @Expose()
+    private final String claimsRequestJson;
+
     @Expose()
     @NonNull
     private final AbstractAuthenticationScheme authenticationScheme;
 
+    @Expose()
+    private final String mamEnrollmentId;
```

Comment:
You want to support MAM?

Reply:
My question then would be that why are you using it in `NativeAuthMsalController`? I thought you earlier mentioned that you are not supporting MAM for Native Auth

Reply:
@SaurabhMSFT Is this addressed?

Reply:
I have removed it